.github/workflows/svelte.yml

```diff
@@ -2,7 +2,7 @@ name: Svelte
 
 on:
   push:
-    branches: ["**"]
+    branches: ["main"]
```

Comment:
@szattila98 What do you think?
Only _main_ will be checked by commits, all other branch will be checked by PRs